src/typescript/frontend/src/styles/globals.css

```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer utilities {
+  .scrollbar::-webkit-scrollbar {
+    width: 0px;
+  }
```

Comment:
Class names should be descriptive (not clear what `scrollbar` would do, whereas something like `scrollbar-none` makes it clear that you’re hiding it)